multi-node/aws/pkg/config/config.go

```diff
@@ -108,9 +114,25 @@ func (c Cluster) Config() (*Config, error) {
 		return nil, fmt.Errorf("failed getting AMI for config: %v", err)
 	}
 
+	//Set logical name constants
+	config.VPCLogicalName = vpcLogicalName
+
+	//Set reference strings
+	config.VPCRef = dynamicResourceRef(vpcLogicalName, config.VPCID)
```

Comment:
It seems like there are only 2 options for dynamicResourceRef. Might suggest just changing this to:

```
if resID != "" {
  config.VPCRef = fmt.Sprintf("%q", resID)
} else {
  config.VPCRef = `{ "Ref" : "VPC"}`
}
```